src/protocolgame.h

```diff
@@ -320,6 +320,7 @@ class ProtocolGame final : public Protocol
 
 		bool debugAssertSent = false;
 		bool acceptPackets = false;
+		bool loggedIn = false;
```

Comment:
Is this unused?

Reply:
@djarek 
`if (!loggedIn && tile->getPosition() == player->getPosition()) {`